pkg/apis/kops/channel.go

```diff
@@ -84,6 +84,10 @@ type ChannelImageSpec struct {
 
 // LoadChannel loads a Channel object from the specified VFS location
 func LoadChannel(location string) (*Channel, error) {
+	if location == "none" {
```

Comment:
why is this needed? I don't see `none` referenced anywhere else

Reply:
This allows for a simple way to opt out of channels.

Reply:
ah ok. It would be helpful to document the different values for `--channel` in the flag's help text (`stable`, `alpha`, `none`, or a URL? any others?) but that could be done in a separate PR.

Reply:
and consider adding `none` here: https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/channel.go#L33 and using it in the string comparison. I'd be hesitant to add an `alpha` const because we don't actually reference the specific value anywhere, we just treat it as a relative path appended to the channel base URL. But maybe it would be helpful for the flag's help text.